package.json

```diff
@@ -17,20 +18,22 @@
     "@babel/register": "^7.8.3",
     "chai": "^4.2.0",
     "chai-http": "^4.3.0",
-    "coveralls": "^3.0.9",
+    "cross-env": "^6.0.3",
     "eslint": "^6.8.0",
     "eslint-config-airbnb": "^18.0.1",
     "eslint-plugin-import": "^2.20.0",
+    "nodemon": "^1.19.4",
     "mocha": "^7.0.0",
-    "nyc": "^15.0.0",
-    "nodemon": "^1.19.4"
+    "nyc": "^15.0.0"
+    "coveralls": "^3.0.9"
```

Comment:
There's is an error in this file, please fix it.

Thanks

Reply:
Resolved

Reply:
It's not resolved, I can still see the error